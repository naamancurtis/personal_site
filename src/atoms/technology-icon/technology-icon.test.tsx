import React from 'react';
import { shallow } from 'enzyme';
import TechnologyIcon from './technology-icon.component';

describe('<TechnologyIcon />', () => {
  it('expect to render component', () => {
    const wrapper = shallow(
      <TechnologyIcon description="test" imgSrc="test" alt="test" />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
