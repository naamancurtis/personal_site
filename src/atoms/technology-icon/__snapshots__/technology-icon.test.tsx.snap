// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TechnologyIcon /> expect to render component 1`] = `
<styled.div>
  <styled.img
    alt="test"
    src="test"
  />
  <styled.div>
    test
  </styled.div>
</styled.div>
`;
